### DISCUSSION TITLE
Quickview available but shows nothing but an icon

### DISCUSSION THREAD

**Discussion by NolanNolan** [Flow-Launcher/Flow.Launcher#1638] [General]
Hi

I just installed todays update version 1.10.0 and saw there is now a preview window, but selecting a file does not show a preview just a icon of a pdf, jpt or docs.

Is that expected or do i miss some setting ?

thanks

Nolan

**Comment by jjw24** [comment]
Hi @NolanNolan , it is still in early stages so currently will only show preview for media files, these are- ".jpg", ".png", ".avi", ".mkv", ".bmp", ".gif", ".wmv", ".mp3", ".flac", ".mp4". So preview showing only an icon for the other file types is normal.

**Comment by NolanNolan** [comment]
ok i see sounds great documents are being supported too - thanks :-)

Nolan

**Comment by hai1120** [comment]
这是来自QQ邮箱的自动回复邮件。
 
你好，你的邮件我已收到，会尽快给您回复，谢谢。

**Comment by jjw24** [comment]
DeborahGarcia5916 and hai1120 message hidden due to spam. Locking thread.
